apps/rule-manager/client/src/ts/components/TagsSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import {useEffect, useState} from "react";
+import {EuiFormRow, EuiComboBox} from "@elastic/eui";
+import React from "react";
+import { MetadataOption } from "./CategorySelector";
+import { PartiallyUpdateRuleData, RuleFormData } from "./RuleForm";
+
+const existingTags = [
```

Comment:
Non-blocking – should this live in 'constants'?

Reply:
Good idea, now done!